.circleci/config.yml

```diff
@@ -1,4 +1,19 @@
 version: 2.1
+
+parameters:
+  GHA_Actor:
+    type: string
+    default: ""
+  GHA_Action:
+    type: string
+    default: ""
+  GHA_Event:
+    type: string
+    default: ""
+  GHA_Meta:
```

Comment:
I propose we call this GHA_PR_NUMBER

Reply:
the GHA params are hard coded into the plugin